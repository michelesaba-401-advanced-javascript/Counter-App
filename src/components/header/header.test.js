import React from 'react';
import renderer from 'react-test-renderer';

import Header from '.';

describe('Header Component', () => {
  it('matches snapshot without props', () => {
    const tree = renderer.create(<Header />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('can shallowly render without props', () => {
    let header = shallow(<Header />);
    expect(header.find('h1').exists()).toBe(true);
  });
});
