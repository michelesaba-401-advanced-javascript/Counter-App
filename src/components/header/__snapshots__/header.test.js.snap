// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot without props 1`] = `
<header>
  <h1>
    Clicker!
  </h1>
</header>
`;
